service-front/module/Application/view/application/pages/address_verification.twig

```diff
@@ -1,6 +1,6 @@
 {% include 'layout/id_check_banner.twig' with details_data %}
-<div class="govuk-width-container">
-    <main class="govuk-main-wrapper">
+<div class="govuk-width-container govuk-!-padding-top-5">
```

Comment:
Possible easier to do this as a follow-up PR because it'll change all the whitespace, but I think every page now starts with `<div class="govuk-width-container govuk-!-padding-top-5">`: could that just be moved into `service-front/module/Application/view/layout/layout.twig` to avoid duplication?